js/index.js

```diff
@@ -0,0 +1,33 @@
+var FizzBuzz = (function () {
+
+  'use strict';
+
+  var _fizzBuzz = function (x, y) {
+    this.wordOne = x || "Fizz";
+    this.wordTwo = y || "Buzz";
+  };
+
+  var createArray = function(min, max, word1, word2, array) {
+    for (var i = min; i <= max; i++) {
+      array.push(i % 15 === 0 ? word1 + word2 : i % 3 === 0 ? word1 : i % 5 === 0 ? word2 : i);
```

Comment:
@pstrum I think I pointed this out to you at a different point, but compound ternary operators are usually a bad thing; they're hard to read and they're brittle. Better would be to abstract this out in to a function that uses a stacked `if/else` statement or even a `switch` statement.